### ISSUE TITLE
Support for css module `@value`?

### ISSUE THREAD

**Issue by fnlctrl** [cuth/postcss-pxtorem#47]
Hi, thanks for this aweseome plugin!
I'm currently using css modules and found that `@value` rules are not being transformed, for example

```css
@value foo: 16px;
.bar {
  font-size: foo;
  line-height: 32px;
}
```
is transformed to
```css
.bar {
  font-size: 16px;
  line-height: 2em;
}
```
where the `@value` variable is missed out. It would be great if `@value` is supported.

**Comment by cuth** [comment]
Is that a PostCSS plugin that makes @value work? I'm wondering if you can just change the order and run this plugin last.

**Comment by fnlctrl** [comment]
The `@value` support is enabled by css-loader with a config `{ modules: true }`. After some digging I found that with this config it applies its own set of plugins for postcss (list can be found [here](https://github.com/webpack-contrib/css-loader/blob/master/lib/processCss.js#L162-L189)"). 
So I guess it's up to css-loader to support passing custom css plugins, and there's probably nothing this plugin can do. Thanks for the reply! Closing this issue for now.
